### ISSUE TITLE
The property 'IsInstalled' cannot be found on this object.

### ISSUE THREAD

**Issue by eknraw** [microsoft/navcontainerhelper#1380]
**Describe the issue**

Running Run-AlPipeline locally is throwing an error during the import of the Test Toolkit using the installTestLibraries option.

**Scripts used to create container and cause the issue**

I'm using the Local-Pipeline.ps1 script created as part of the Helloworld Business Central Apps project on Azure Devops. 

**Full output of scripts**
```
  _____                            _   _               _______       _     _______          _ _    _ _   
 |_   _|                          | | (_)             |__   __|     | |   |__   __|        | | |  (_) |
   | |  _ __ ___  _ __   ___  _ __| |_ _ _ __   __ _     | | ___ ___| |_     | | ___   ___ | | | ___| |_
   | | | '_ ` _ \| '_ \ / _ \| '__| __| | '_ \ / _` |    | |/ _ \ __| __|    | |/ _ \ / _ \| | |/ / | __|
  _| |_| | | | | | |_) | (_) | |  | |_| | | | | (_| |    | |  __\__ \ |_     | | (_) | (_) | |   <| | |_
 |_____|_| |_| |_| .__/ \___/|_|   \__|_|_| |_|\__, |    |_|\___|___/\__|    |_|\___/ \___/|_|_|\_\_|\__|
                 | |                            __/ |
                 |_|                           |___/
% : The property 'IsInstalled' cannot be found on this object. Verify that the property exists.
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.10\ObjectHandling\Import-TestToolkitToNavContainer.ps1:127 char:21
+         $appFiles | % {
+                     ~~~
    + CategoryInfo          : NotSpecified: (:) [ForEach-Object], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict,Microsoft.PowerShell.Commands.ForEachObjectCommand
...
```

I think it might be the license warning that's causing the problem.   I'll experiment a bit more.

![image](https://user-images.githubusercontent.com/5561726/96524634-6d8a8900-123e-11eb-8b57-e7d3f157555d.png)

**Comment by eknraw** [comment]
Yes, that appears to be the issue.  I removed the WARNING but I'm not sure that's a great way to do it.  I can open up a PR if you'd like to handle it like that, possibly differently if you have suggestions. 

Line 149 of Import-TestToolkitToNavContainer.ps1:

`$tenantAppInfo = $tenantAppInfo -replace "WARNING: This license is not compatible with this version of Business Central." , ''`

**Comment by freddydk** [comment]
You are right.
The fix I have used in other places (and will add here as well) is:
```
            $tenantAppInfo = Invoke-ScriptInBCContainer -containerName $containerName -scriptblock { Param($appFile, $tenant)
                $navAppInfo = Get-NAVAppInfo -Path $appFile
                (Get-NAVAppInfo -ServerInstance $serverInstance -Name $navAppInfo.Name -Publisher $navAppInfo.Publisher -Version $navAppInfo.Version -tenant $tenant -tenantSpecificProperties)
            } -argumentList $appFile, $tenant | Where-Object { $_ -isnot [System.String] }
```
Thanks for catching this.

**Comment by freddydk** [comment]
Shipped in 1.0.10-preview224

**Comment by freddydk** [comment]
Fix shipped in 1.0.10
